### ISSUE TITLE
Add ctsm_sci test list that would be used for releases and making sure important resolutions run well

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#1142]
We think it would be good to have a "ctsm_sci" test list that would have the list of important resolutions/configurations used for CTSM science. I think this would be f09_g17, g19_g17, and f45. It should also have a NWP configuration. Anything that we list as "scientifically supported" should be in this test list. Probably would all be for production compiler settings, maybe just one test at debug to make sure there aren't problems with it.

This would also have the resolutions that are critical for CAM (so C96 and ARCTIC, CONUS, ARTICGRIS grids). It would just be for the standard compiler on the main production machine (cheyenne_intel right now).

This test list would be run supplementary to the regular aux_clm test that would be used for normal tags. This would be run for releases, or important tags that we want to make sure the science resolutions are running. We could move some of the tests we have in aux_clm to this test list, to save time in the normal test list. This would be thought of as a supplementary test list that's run in addition to the standard test list done for tags. As such it doesn't need to test the same things already done for aux_clm.

This relates to these issues that talk about testing...
#1139 
#1141

**Comment by ekluzek** [comment]
I've started this in #1150. I just included the list of compsets that have "science_supported" clicked. So only f09 and f19. There are some issues that I think are just due to known problems. So I'm bringing it in with the following fails...

SMS_Ld5.f09_g17.IHistClm45BgcCruGs.cheyenne_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm45BgcGs.cheyenne_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm50Sp.cheyenne_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm50SpCru.cheyenne_intel.clm-default
SMS_Ld3.f09_g17.I1850Clm50BgcCropCru.izumi_intel.clm-default		
SMS_Ld5.f09_g17.I1850Clm50Bgc.izumi_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm45BgcCruGs.izumi_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm45BgcGs.izumi_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm50Sp.izumi_intel.clm-default		
SMS_Ld5.f09_g17.IHistClm50SpCru.izumi_intel.clm-default	

Since there are scientists that run on izumi it would be good to have f09 and f19 tests there. We wouldn't add the CAM grids to izumi though.
